packages/teams-js/src/public/media.ts

```diff
@@ -756,6 +756,9 @@ export namespace media {
   /**
    * Select an attachment using camera/gallery
    *
+   * @remarks
```

Comment:
>remarks [](http://example.com/codeflow?start=6&length=7)

Would it be more appropriate to put this on the `media` namespace rather than this one function? Presumably this applies to other functions in this file?